console/src/main/java/com/alibaba/nacos/console/config/ConsoleConfig.java

```diff
@@ -51,6 +52,7 @@ public class ConsoleConfig {
      */
     @PostConstruct
     public void init() {
+        EnvUtil.customEnvironment();
```

Comment:
I has a problem about this. Should we move this to nacos-core module?

Reply:
The purpose of this is to execute after loading application.properties and before initializing the controller.

I haven't yet found where nacos-core can achieve this effect.

If you do, you can move it to nacos-core.

Reply:
I'm not sure, I just think about the feature will no effect when users not start with console. The custom method will not be called.

Reply:
Okay, I'll try .

Reply:
down